bitnami/kafka/Chart.yaml

```
@@ -29,4 +29,4 @@ name: kafka
 sources:
   - https://github.com/bitnami/bitnami-docker-kafka
   - https://kafka.apache.org/
-version: 15.0.4
+version: 15.0.5
```

You're including a new parameter, then you need to bump the minor version
```suggestion
version: 15.1.0
```